glustercli/cmd/rebalance.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/restclient"
+	rebalance "github.com/gluster/glusterd2/plugins/rebalance/api"
+)
+
+const (
+	helpRebalanceCmd       = "Gluster Rebalance"
+	helpRebalanceStartCmd  = "Start rebalance session for gluster volume"
+	helpRebalanceStatusCmd = "Status of rebalance seesion"
+	helpRebalanceStopCmd   = "Stop rebalance session"
+)
+
+var (
+	flagRebalanceStartCmdForce     bool
+	flagRebalanceStartCmdFixLayout bool
+)
+
+var rebalanceCmd = &cobra.Command{
+	Use:   "volume rebalance",
```

Comment:
where ever its required place add ```Long``` command to give more information about this CLI command to users

Reply:
ok